apps/judicial-system/digital-mailbox-api/src/app/modules/cases/models/subpoena.response.ts

```diff
@@ -59,31 +76,48 @@ export class SubpoenaResponse {
 
     const waivedRight = defendantInfo?.defenderChoice === DefenderChoice.WAIVE
     const hasDefender = defendantInfo?.defenderName !== undefined
+    const subpoena = defendantInfo?.subpoenas ?? []
+    const subpoenaAcknowledged = subpoena[0]?.acknowledged ?? false
```

Comment:
The subpoenas are already ordered by date in the backend so this would be redundant, but I might do it anyways just to account for any changes in the backend in the future.